scipy/signal/tests/test_fir_filter_design.py

```diff
@@ -481,6 +481,70 @@ def test_fs_validation(self):
         with pytest.raises(ValueError, match="Sampling.*single scalar"):
             remez(11, .1, 1, fs=np.array([10, 20]))
 
+
+class TestRemezord:
+    """
+    Test examples taken from:
+    http://www.ece.northwestern.edu/local-apps/matlabhelp/toolbox/signal/remezord.html
```

Comment:
I don't see these numbers on that page, so assuming the reference values were computed in MATLAB or Octave can you add that to the comment/doc here? Like:

```suggestion
    http://www.ece.northwestern.edu/local-apps/matlabhelp/toolbox/signal/remezord.html
    with reference values computed in MATLAB.
```
At least in MATLAB I get:
```MATLAB
>> format compact
>> format long
>> rp = 3;          % Passband ripple
rs = 40;         % Stopband ripple
fs = 2000;       % Sampling frequency
f = [500 600];   % Cutoff frequencies
a = [1 0];       % Desired amplitudes
% Compute deviations
dev = [(10^(rp/20)-1)/(10^(rp/20)+1)  10^(-rs/20)]; 
[n,fo,ao,w] = remezord(f,a,dev,fs);
>> n, fo, ao, w
n =
    22
fo =
                   0
   0.500000000000000
   0.600000000000000
   1.000000000000000
ao =
     1
     1
     0
     0
w =
   1.000000000000000
  17.0997357343619040
```

Also the Example 2 isn't in there, but locally I ran it in MATLAB and get the following, can it also become a test case?
```MATLAB
>> [n,fo,ao,w] = remezord([1500 2000],[1 0],[0.01 0.1],8000 );
>> n, fo, ao, w
n =
    21
fo =
         0
    0.3750
    0.5000
    1.0000
ao =
     1
     1
     0
     0
w =
    10
     1
```

Reply:
Added Example 2:

https://github.com/scipy/scipy/pull/20983/files#diff-bb7c811e81932922f4a8964acc03ac3633a245fbc07a6120d4c814d51d2b810fR550-R559